login.html

```diff
@@ -0,0 +1,38 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class= "login-container">
+    <img class= "wolox-logo"
+      src= "https://trello-attachments.s3.amazonaws.com/607457e8ad4162814da41ef0/607457e8ad4162814da41f08/x/ab567d29d6f06a584a4102325415d39f/LogoWolox.png"
+      alt= "wolox logo"
+    >
+    <form>
+      <p class= "text"> Email </p>
+      <input class= "credentials-box">
+      </input>
+      <p class= "text"> Password </p>
+      <input class= "credentials-box">
+      </input>
+      <div class= "button">
+        <button href= "book-list.html" class = "action-button">
+          Login  
+        </button>
+      </div>
+      <div class="separator"></div>
+      <div class= "button">
+        <button class= "redirection-button">
+          Sign Up   
+        </button>
+      </div>
+    </form>
```

Comment:
The button can be moved outside the form, and if you add a border-bottom to it you can remove the element
`<div class="separator"></div>`